Budget Manager/task/src/budget/Menu.java

```diff
@@ -0,0 +1,75 @@
+package budget;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    public Menu() {
+    }
+
+    private static void print(String string) {
+        System.out.print(string);
+    }
+
+    public static String getOptionFromInputSortMenu() {
+        print("\nChoose the type of purchase" +
+                "\n1) Food" +
+                "\n2) Clothes" +
+                "\n3) Entertainment" +
+                "\n4) Other" +
+                "\n");
+        return getUserInput();
+    }
+
+    public static String getOptionFromSortMenu() {
+        print("\nHow do you want to sort?" +
+                "\n1) Sort all purchases" +
+                "\n2) Sort by type" +
+                "\n3) Sort certain type" +
+                "\n4) Back" +
+                "\n");
+        return getUserInput();
+    }
+
+    public static String getOptionFromPrintTypeMenu() {
+        print("\nChoose the type of purchases" +
+                "\n1) Food" +
+                "\n2) Clothes" +
+                "\n3) Entertainment" +
+                "\n4) Other" +
+                "\n5) All" +
+                "\n6) Back" +
+                "\n");
+        return getUserInput();
+    }
+
+    public static String getOptionFromInputTypeMenu() {
+        print("\nChoose the type of purchase" +
+                "\n1) Food" +
+                "\n2) Clothes" +
+                "\n3) Entertainment" +
+                "\n4) Other" +
+                "\n5) Back" +
+                "\n");
+        return getUserInput();
+    }
+
+    public static String getOptionFromGeneralMenu() {
+        print("Choose your action:" +
+                "\n1) Add income" +
+                "\n2) Add purchase" +
+                "\n3) Show list of purchases" +
+                "\n4) Balance" +
+                "\n5) Save" +
+                "\n6) Load" +
```

Comment:
Al deze lijsten zorgen bij mij voor een gevoel: gebruik enums. Zeker als ik kijk hoe het verder gebruikt wordt. Naar enums kan je vrij makkelijk parsen vanuit input, daarnaast kan je daarin methods definieren die voor die enum groep horen en meteen voor alles geld, zoals bijv. een toString(). Ook heeft het als voordeel dat door heel je code duidelijk is welk state je aan het gebruiken bent ipv door te kijken naar een nr. Nog een plus punt is dat een nieuw scenario makkelijker toe te voegen is met enums. Maar goed groot deel is smaak ;)